src/test/java/pageObjects/TransferFunds.java

```diff
@@ -0,0 +1,92 @@
+package pageObjects;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.List;
+
+public class TransferFunds {
+
+
+    @FindBy(xpath = "//a[contains(text(),'Transfer Funds')]")
+    public WebElement transferfund;
+
+
+    @FindBy(xpath = "//input[@id='amount']")
```

Comment:
input >*?

Reply:
here when i use * it reffer me 2 buttons
<img width="1092" alt="Screenshot 2022-12-20 at 9 50 32 AM" src="https://user-images.githubusercontent.com/119111500/209266618-1b5191eb-867d-4197-9b3d-51566705c654.png">